packages/core/core-flows/src/cart/workflows/complete-cart.ts

```diff
@@ -298,6 +260,37 @@ export const completeCartWorkflow = createWorkflow(
         }
       })
 
+      const promotionUsage = transform(
```

Comment:
comment: @riqwan, would it be OK to move this to a parallelized step? It used to be a separate one, if you look at the diff.

Reply:
looks like adjustments have already been settled by then, should be ok 🤞🏻